src/main/java/recipe/RecipeRequestModel.java

```diff
@@ -26,13 +26,13 @@ public RecipeRequestModel(String recipe_name, String procedure, String cuisine,
         this.creator = creator;
     }
 
-    public String get_procedure(){return this.procedure;}
-    public String get_recipe_name(){return this.recipe_name;
+    public String getProcedure(){return this.procedure;}
+    public String getRecipeName(){return this.recipe_name;
     }
-    public String get_cuisine(){return this.cuisine;}
-    public int get_calories(){return this.calories;}
-    public int get_time(){return this.time;}
-    public int get_difficulty(){return this.difficulty;}
-    public ArrayList<String> get_ingredients(){return this.ingredients;}
-    public String get_creator(){return this.creator;}
+    public String getCuisine(){return this.cuisine;}
+    public int getCalories(){return this.calories;}
+    public int getTime(){return this.time;}
+    public int getDifficulty(){return this.difficulty;}
+    public ArrayList<String> getIngredients(){return this.ingredients;}
+    public String getCreator(){return this.creator;}
```

Comment:
Looks great! To double-check, since creator is a String, would this be the username of the User who created this recipe?